tokenizer_test.go

```diff
@@ -227,8 +364,8 @@ func BenchmarkEncodeNTimes(b *testing.B) {
 	expected := []uint32{2829, 4419, 14523, 2058, 1996, 13971, 3899}
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		tokens, _ := tk.Encode("brown fox jumps over the lazy dog", false)
-		assert.Equal(b, expected, tokens)
+		ids, _ := tk.Encode("brown fox jumps over the lazy dog", false)
```

Comment:
do you mind running before and after benchmarks?

Reply:
Sure. I will post the results in the PR.

Reply:
done